scripts/index-demo-page.js

```diff
@@ -251,13 +251,15 @@
     stateEls.partial.addEventListener('change', function(event) {
       saveState();
 
-      window.videojs.options = window.videojs.options || {};
-      window.videojs.options.vhs = window.videojs.options.vhs || {};
-      window.videojs.options.vhs.handlePartialData = event.target.checked;
+      // reload the player and scripts
+      stateEls.minified.dispatchEvent(newEvent('change'));
```

Comment:
I tried to use this code for the buffer water line option and noticed that it didn't really work in all cases. So I updated them both.